cache/bug_excerpt.go

```diff
@@ -31,10 +34,13 @@ func NewBugExcerpt(b bug.Interface, snap *bug.Snapshot) *BugExcerpt {
 		EditLamportTime:   b.EditLamportTime(),
 		CreateUnixTime:    b.FirstOp().GetUnixTime(),
 		EditUnixTime:      snap.LastEditUnix(),
+		Title:             snap.Title,
 		Status:            snap.Status,
 		Author:            snap.Author,
 		Labels:            snap.Labels,
+		NoOfComments:      len(snap.Comments),
 		CreateMetadata:    b.FirstOp().AllMetadata(),
+
```

Comment:
Please don't add this kind of non-useful lines ;)

Reply:
Done :+1: